R-package/src/xgboost_R.h

```diff
@@ -174,13 +202,37 @@ XGB_DLL SEXP XGDMatrixGetDataAsCSR_R(SEXP handle);
  */
 XGB_DLL SEXP XGBoosterCreate_R(SEXP dmats);
 
+/*!
+ * \brief copy information about features from a DMatrix into a Booster
+ * \param booster R 'externalptr' pointing to a booster object
+ * \param dmat R 'externalptr' pointing to a DMatrix object
+ */
+XGB_DLL SEXP XGBoosterCopyInfoFromDMatrix(SEXP booster, SEXP dmat);
+
+/*!
+ * \brief handle R 'externalptr' holding the booster object
+ * \param field field name
+ * \param features features to set for the field
```

Comment:
I think this is feature or features based on the dispatching logic you have in the R C function.

Reply:
But that's the "field" content.